backend/src/main/resources/migrations.xml

```diff
@@ -24,7 +24,26 @@
             <column name="idea_id" type="BIGINT(20)">
                 <constraints nullable="false"/>
             </column>
-            <column name="invite_status" type="TINYINT(3)"/>
+            <column name="idea_invitation_id" type="BIGINT(20)">
+                <constraints nullable="false"/>
```

Comment:
I'm concerned about making this `not null`. That means that all collaborators need an invitation to exist, so in order to have a collaborator we need an invitation first.

Lemme continue reviewing before changing it.